base/staticdata.jl

```diff
@@ -73,6 +73,51 @@ end
 
 get_require_world() = unsafe_load(cglobal(:jl_require_world, UInt))
 
+function gen_staged_sig(def::Method, mi::MethodInstance)
```

Comment:
This file feels like a slightly strange place to define this, but obviously not really a problem